rpcs3/Emu/Cell/Modules/sceNp.cpp

```diff
@@ -3957,8 +3957,9 @@ error_code sceNpManagerGetAccountRegion(vm::ptr<SceNpCountryCode> countryCode, v
 	}
 
 	memset(countryCode.get_ptr(), 0, sizeof(SceNpCountryCode));
-	countryCode->data[0] = 'u';
-	countryCode->data[1] = 's';
+	const auto s_country_code = nph.get_country_code();
+	countryCode->data[0] = s_country_code.data[0];
+	countryCode->data[1] = s_country_code.data[1];
```

Comment:
Remove the memset above and use:
```suggestion
	np::string_to_country_code(nph.get_country_code(), *countryCode);
```

Reply:
It's throwing an error when applied:
![image](https://github.com/user-attachments/assets/c4383f5f-ae6f-45ec-a967-55305e8152a7)

Reply:
remove all of the code in np_handler and use:
```np::string_to_country_code(g_cfg.net.country.to_string(), *countryCode);```

Reply:
Actrually, also remove string_to_country_code and just copy its code here.

Reply:
OK, will do.